ui/components/ErrorBoundary.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from "react";
+import Page from "./Page";
+
+export default class ErrorBoundary extends React.Component<
```

Comment:
Oh cool! I've not used this before, just familiarised myself now with the docs on the react website. Perhaps I'd rewrite to follow the latest syntax - not a big deal though

Reply:
As I understand it, this is the latest syntax.

https://reactjs.org/docs/error-boundaries.html

I don't think we can do a function component for this, if that's what you mean.

Reply:
Yup exactly that's what I was thinking. Oki np, good to know